package.json

```diff
@@ -20,15 +20,41 @@
     "start": "ember serve",
     "test": "npm-run-all lint:* test:*",
     "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each"
+    "test:ember-compatibility": "ember try:each",
+    "prepack": "ember ts:precompile",
+    "postpack": "ember ts:clean"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.22.1"
+    "ember-cli-babel": "^7.22.1",
+    "ember-cli-typescript": "^5.1.0"
```

Comment:
As mentioned in my general comment, I'd prefer not to use ember-cli-typescript.

Reply:
Why resolve?

Reply:
Because I’ll need the addon v2 PR to land first before making any meaningful changes to this one